next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -43,28 +43,24 @@ export function MultiCheckboxFacet({
           const checked = option.selected;
           const value = option.value as FieldValue;
           return (
-            <li key={`${getFilterValueDisplay(option.value)}`}>
-              <div className="flex items-center">
-                <Checkbox
-                  data-transaction-name={`facet - ${label}`}
-                  id={`example_facet_${label}${getFilterValueDisplay(
-                    option.value
-                  )}`}
-                  checked={checked}
-                  onClick={() => (checked ? onRemove(value) : onSelect(value))}
-                />
-                <label
-                  className="ml-2 text-sm text-steelgray"
-                  htmlFor={`example_facet_${label}${getFilterValueDisplay(
-                    option.value
-                  )}`}
-                >
-                  {getFilterValueDisplay(option.value)}{" "}
-                  <span className="text-steelgray">
-                    ({option.count.toLocaleString("en")})
-                  </span>
-                </label>
-              </div>
+            <li
+              key={`${getFilterValueDisplay(option.value)}`}
+              className="flex items-center"
+            >
+              <Checkbox
+                data-transaction-name={`facet - ${label}`}
+                aria-label={`${t("filter-by")} ${getFilterValueDisplay(
+                  option.value
+                )} (${option.count.toLocaleString("en")})`}
```

Comment:
I think we can use `aria-labelledby` and point it to the text just after the checkbox, rather than duplicating it? (Of course, we'd remove the `aria-hidden` from it in that case)

The "filter by" part can be removed if we add "filter by" to the initial label as I suggested in an earlier comment - this would also prevent repeating "filter by" on every option, because the user would already know it's for filtering.